frontend/app/src/components/history/events/HistoryEventsView.vue

```diff
@@ -330,6 +331,16 @@ function onShowDialog(type: 'decode' | 'protocol-refresh'): void {
     set(protocolCacheStatusDialogOpen, true);
 }
 
+async function redecodePageTransactions(): Promise<void> {
+  const evmEvents = currentPageEvents.value.filter(isEvmEvent);
+  const { pullAndRedecodeTransaction, fetchUndecodedTransactionsStatus } = useHistoryTransactionDecoding();
```

Comment:
never call composable inside function like this, as this will be called everytime the function called. Move it outside

Reply:
Also this is redundant, we have called it outside